bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -0,0 +1,97 @@
+# BoschSHC Binding
+
+Binding for the Bosch Smart Home Controller:
+
+## Supported Things
+
+ - Bosch In-Wall switches
+ - Bosch Smart Plugs (use "in-wall-switch" thing too).
+ - Bosch TwinGuard smoke detector
+ - Bosch Window/Door contacts
+ - Bosch Motion Detector
+ - Bosch Shutter Control in-wall
+ - Bosch Thermostat
+ - Bosch Climate Control
+
```

Comment:
Can you add a table with all Channels? Example: https://www.openhab.org/addons/bindings/telegram/#channels

Reply:
Added tables with the channels for each thing